docs/concepts/controllers/statefulsets.md

```diff
@@ -0,0 +1,171 @@
+---
+assignees:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+---
+
+{% capture overview %}
+**StatefulSets are a beta feature in 1.5. This feature replaces the 
+Pet Sets feature from 1.4. Users of Pet Sets are referred to the 1.5 
+[Upgrade Guide](/docs/tasks/stateful-set/upgrade-from-petsets-to-stateful-sets/)
+for further information on how to upgrade existing Pet Sets to StatefulSets.**
+
+A StatefulSet is a Controller that provides a unique identity to its Pods. It provides
+guarantees about the ordering of deployment and scaling.
+{% endcapture %}
+
+{% capture body %}
+
+### When to Use a StatefulSet
+StatefulSets are valuable for applications that require one or more of the 
+following.
+
+* Stable, unique network identifiers.
+* Stable, persistent storage.
+* Ordered, graceful deployment and scaling.
+* Ordered, graceful deletion and termination.
+
+In the above, by stable, we mean persistent across Pod (re) schedulings.
```

Comment:
Avoid "we" constructs. Rephrase:

"Both the unique network identifiers and the persistent storage associated with a StatefulSet are "stable" in the sense that they are persistent across Pod scheduling or re-scheduling."